hbase-server/src/test/java/org/apache/hadoop/hbase/rsgroup/TestRSGroupsAdmin1.java

```diff
@@ -468,4 +468,31 @@ public boolean evaluate() throws Exception {
     // Cleanup
     TEST_UTIL.deleteTable(tn1);
   }
+
+  @Test
+  public void testRenameRSGroup() throws Exception {
+    // Add rsgroup, and assign 2 servers and a table to it.
+    RSGroupInfo oldgroup = addGroup("oldgroup", 2);
+    TableName tb = TableName.valueOf("testRename");
+    TEST_UTIL.createTable(tb, "tr");
+    ADMIN.setRSGroup(Sets.newHashSet(tb), oldgroup.getName());
+    Thread.sleep(500);
```

Comment:
Instead of sleep(), we can use `HBASE_TESTING_UTILITY.waitFor`:
e.g.
```
HBASE_TESTING_UTILITY.waitFor(1000,
      () -> {
          oldgroup = ADMIN.getRSGroup(oldgroup.getName());
          return oldgroup.getServers().size() == 2 && ADMIN.getRSGroup(tb).getName().equals(oldgroup.getName());
      });
```
This way we will ensure, we wait for specific period of time until our predicate returns true.

Reply:
fixed